R/parse_compat.R

```diff
@@ -50,12 +105,17 @@ parse_compat_report <- function(exprs, action, call) {
   i <- !vlapply(exprs, function(x) is.null(x$compat))
   if (action != "silent" && any(i)) {
     description <- c(
-      user = "Replace calls to 'user()' with 'parameter()'")
-    type <- vcapply(exprs[i], function(x) x$compat$type)
+      user = "Replace calls to 'user()' with 'parameter()'",
+      distribution = paste(
+        "Replace calls to r-style random number calls (e.g., 'rnorm()')",
+        "with mcstate2-stye calls (e.g., 'Normal()')"))
+
+    type <- lapply(exprs[i], function(x) x$compat$type)
+    err <- unlist0(type)
 
     detail <- NULL
-    for (t in intersect(names(description), type)) {
-      j <- i[type == t]
+    for (t in intersect(names(description), err)) {
+      j <- which(i)[vlapply(type, function(x) t %in% x)]
```

Comment:
On line 125, should "x" and "v" be the other way round - red cross for original, and green tick for the correct updated one? `context_t` with the test case below ends up being:-
```
                          x                           v 
"update(a) <- Normal(a, 1)"  "update(a) <- rnorm(a, 1)" 
```

Reply:
oops, you are right - fixed now